src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -521,6 +521,12 @@ public class FocusRequestArgs : EventArgs
         {
             get => swigCPtr;
         }
+
+        /// <summary>
+        /// A flag to check if it is already disposed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Hello, please add this~

**_/// \<remarks>Hidden API: Only for inhouse or developing usage. The behavior and interface can be changed anytime.\</remarks>_**